app.rb

```diff
@@ -11,143 +11,66 @@ def initialize()
     @rentals = []
   end
 
-  def app_options
-    puts '1 - List all books'
-    puts '2 - List all people'
-    puts '3 - Create a person'
-    puts '4 - Create a book'
-    puts '5 - Create a rental'
-    puts '6 - List all rentals for a given person id'
-    puts '7 - Exit'.colorize(color: :red).underline
-    puts "\n\n"
-  end
+  @app_options = [
+    '[0]  =>  List all books',
+    '[1]  =>  List all people',
+    '[2]  =>  List all students',
+    '[3]  =>  List all teachers',
+    '[4]  =>  Create a person',
+    '[5]  =>  Create a book',
+    '[6]  =>  Create a rental',
+    '[7]  =>  List all rentals for a given person id',
+    '[8]  =>  List all rentals for a given book',
+    '[9]  =>  Exit'
+  ]
```

Comment:
- [ ] In my local pc I can't run your project it seems it has some problems with the colorize file so make sure you import it right? but after deleting all the code related with this colorize it also can't run


## You can keep your code functions inside app.rb files 💪🏻🚀
As you can see it shows me this error so kindly work on it so the app can run: 
![image](https://user-images.githubusercontent.com/58553711/194976826-a74c6c92-4b52-482e-a023-aee89685f26b.png)


![image](https://user-images.githubusercontent.com/58553711/194976520-d8f21908-27d3-43bf-a5d9-40df361188ff.png)

Reply:
@Reem-lab The problem is that rubocop does not allow the usage of class variables which is what I use to store the options, however, I have refactored this piece of code to be a method and now working as expected :heavy_check_mark:

**Note:** The colorize file is imported in the right way, this was not the problem :+1: 

![image](https://user-images.githubusercontent.com/92395484/195089089-7640cf5b-85fd-4179-bef1-63b59f363afa.png)